scripts/docker-compose.yml

```diff
@@ -187,3 +188,56 @@ services:
       volumes:
           - ./prometheus.yml:/etc/prometheus/prometheus.yml
       image: prom/prometheus
+  debezium:
+    image: quay.io/cloudservices/xjoin-kafka-connect-strimzi
+    restart: always
+    entrypoint: /opt/kafka/kafka_connect_run.sh
+    volumes:
+      - ./xjoin-config/log4j.properties/:/opt/kafka/custom-config/log4j.properties:Z
+    environment:
```

Comment:
With my newbie hands, do we have any standard documentation on which all environment vars are mandatory?

Reply:
Same, Copied from EE. Here is the document - https://debezium.io/documentation/reference/stable/development/engine.html#engine-properties